theme/themes/pxt/views/card.overrides

```diff
@@ -110,7 +110,7 @@
     border-top-right-radius: @borderRadius;
 }
 .ui.cardimage {
-    background-color: @secondaryColor;
+    background-color: @grey;
```

Comment:
? unrelated changes?

Reply:
no, related to Filexplorer changes. (Accessibility)